_apidocs/searchgov-results.md

```diff
@@ -99,6 +121,16 @@ Sites indexed via sitemaps or crawling will use the `/i14y` endpoint. Because mo
       | snippet          | Summary of the document
       | publication_date | Publication date of the document (not available on commercial results)
 
+      **If `include_facets=true` in your request**, the following fields may show as well. We only display fields with content, so some web results may not have all fields indicated here.
```

Comment:
This reads oddly to me, maybe "If you set `includes_facets=true` in your request"

"some web results" seems like individual results to me, how about "some results sets"?

Reply:
Hm, it is individual results though - if one result has `content_type` indicated but another doesn't, only the one with a non-null content type would have that field returned in the result object.

Reply:
updated with your language